docs/pages/blog/mui-core-v5.md

```diff
@@ -0,0 +1,560 @@
+
+### Migration from Enzyme to Testing Library
+
+- The value? Makes it easier for developers to copy the source, and maintainers to make sure the components are easy to test
+
+### TypeScript migration
+
+- Share our progress https://github.com/mui-org/material-ui/issues/15984
+- Covered a bit in https://material-ui.com/blog/2020-q3-update/
+- The value? Helps keeping the types as accurate as possible
+
+### Strict Mode support
+
+- Docs, test, etc. now runs in StrictMode
+- Warn that `@material-ui/styles` is not compatible
+
+## A new product line: X
+
+- What are the new product separation
+- What this product line is about
+- Why does it even exist
+- Present the new Data Grid component, hopefully, released as v5.0.0 at the same time.
+
+## New components
+
+This release comes with eight new components!
+
+### Improved Grid
+
+The development of the Grid was mostly put on hold for the last 3 years, blocked by the size of the statically generated CSS with JSS.
+The [migration to emotion](#migration-from-jss-to-emotion) has unlocked the following, frequently requested, changes:
+
+We have added support for [row & column](/components/grid/#row-amp-column-spacing) spacing:
+
+```jsx
+<Grid container rowSpacing={1} columnSpacing={2} />
+```
+
+We have added support for [responsive values](/components/grid/#responsive-values) on all the props:
+
+```jsx
+<Grid container spacing={{ xs: 2, md: 3 }} />
+```
+
+We have added support for a different [number of columns](/components/grid/#columns) than 12:
+
+```jsx
+<Grid container columns={16}>
+```
+
+We have added an alternative implementation that uses [CSS grid](/components/grid/#css-grid-layout):
+
+```jsx
+<Box display="grid" gridTemplateColumns="repeat(12, 1fr)" gap={2}>
+  <Box gridColumn="span 8">
+    <Item>xs=8</Item>
+  </Box>
+  <Box gridColumn="span 4">
+    <Item>xs=4</Item>
+  </Box>
+  <Box gridColumn="span 4">
+    <Item>xs=4</Item>
+  </Box>
+  <Box gridColumn="span 8">
+    <Item>xs=8</Item>
+  </Box>
+</Box>
+```
+
+### More Material Design icons
+
+The Material Design team at Google has released 600 new icons in five different themes since we released v4.
+We have made them [available](/components/material-icons/) in the `@mui/icons-material` package.
+
+### Stack
+
+We have introduced a new `<Stack>` component.
+It handles one-dimensional layouts, it's similar to how Figma handles auto-layout.
+
+<a href="/components/stack/"><img src="/static/blog/mui-core-v5/stack.png" alt="" style="width: 532px; margin-bottom: 16px;" /></a>
+
+> Note that you might already be using `<Box display="flex" gap={1}>` to solve the same problem.
+> However [browser support](https://caniuse.com/flexbox-gap) for the flexbox `gap` CSS property is lacking in Safari.
+
+You can find [more details](/components/stack/) in the documentation.
+
+### Promotion from the lab
+
+We have promoted six component from the lab, after over two years iterating on feedback:
```

Comment:
```suggestion
We have moved six components from the lab to the main component package, after over two years iterating on feedback:
```

At first read I did not understand whant "promoting" meant

Reply:
Haha!